muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/discussion/body/application/modify-area-dialog.tsx

```diff
@@ -98,31 +98,37 @@ class DiscussionModifyArea extends SessionStateComponent<DiscussionModifyAreaPro
     
     let content = (closeDialog: ()=>any) => [
       (
-      <div className="container container--new-discussion-area-title">          
-        <input key="1" type="text" className="form-field form-field--new-discussion-area-name"
-        placeholder={this.props.i18n.text.get('plugin.discussion.createarea.name')}
-        value={this.state.name} onChange={this.onNameChange} autoFocus/>
+      <div className="container container--new-discussion-area-title">    
+      <div className="environment-dialog__form-element--wrapper">  
+        <div className="environment-dialog__form-element-label">{this.props.i18n.text.get('plugin.discussion.createarea.name')}</div>          
+          <input key="1" type="text" className="environment-dialog__form-element environment-dialog__form-element--new-discussion-area-name"
+          placeholder={this.props.i18n.text.get('plugin.discussion.createarea.name')}
+          value={this.state.name} onChange={this.onNameChange} autoFocus/>
+        </div>
       </div>    
       
       ),
 
       (
        <div className="container container--new-discussion-area-description">          
-         <textarea key="2" className="form-field form-field--new-discussion-area-description"
+         <div className="environment-dialog__form-element--wrapper">  
```

Comment:
If we use modifiers we need to include the base element class name too as modifier is an extra class name and it cannot be the only class name of the html element.